### PR TITLE
Reference updates

### PR DESCRIPTION
**Pull request by matkuliak**
## Summary of the changes / Why this is an improvement
All of the recent UI updates added to reference

## Checklist

 - [x] [CLA](https://crate.io/community/contribute/cla/) is signed

### CODE REVIEW

File: docs/overview.rst [hunk 1]
```diff
@@ -221,27 +206,24 @@ Overview
...
 
-* **Version**: This indicates the version number of CrateDB the cluster is
-  running.
+* **CPU metrics**: Current CPU utilization of the cluster.
 
-* **Storage metrics**: Used and overall storage of the cluster.
+* **Number of nodes**: Number of nodes in the cluster.
 
-* **Date created**: The day of the original deployment of the cluster.
+* **RAM metric**: Amount of memory of your cluster.
```
Comment (SStorm) [inline docs/overview.rst:219]:
```suggestion
* **RAM metric**: Percentage of ram used in each node on average. The sparkline shows the trend for the last hour.
```

File: docs/overview.rst [hunk 2]
```diff
@@ -221,27 +206,24 @@ Overview
...
-
 * **Region**: Name of the region where the cluster is deployed.
 
 * **Plan**: This shows what subscription plan the cluster is running on. For
   more information on our plans, see the documentation on
   `subscription plans`_.
 
-* **Version**: This indicates the version number of CrateDB the cluster is
-  running.
+* **CPU metrics**: Current CPU utilization of the cluster.
```
Comment (SStorm) [inline docs/overview.rst:215]:
```suggestion
* **CPU metrics**: Average CPU utilization on average per node. The sparkline shows the trend for the last hour.
```

File: docs/overview.rst [hunk 3]
```diff
@@ -221,27 +206,24 @@ Overview
...
 
-* **Storage metrics**: Used and overall storage of the cluster.
+* **Number of nodes**: Number of nodes in the cluster.
 
-* **Date created**: The day of the original deployment of the cluster.
+* **RAM metric**: Amount of memory of your cluster.
 
-* **CPU metrics**: Current CPU utilization of the cluster.
+* **Storage metrics**: Used and overall storage of the cluster.
```
Comment (SStorm) [inline docs/overview.rst:221]:
```suggestion
* **Storage metrics**: Used and overall storage of the cluster. The sparkline shows the trend for the last hour.
```

File: docs/overview.rst [hunk 4]
```diff
@@ -264,81 +246,27 @@ Admin UI
...
-  snippets that can be used in the SQL Console of your Admin UI. The first
-  snippet will create a "nyc_taxi" table, and the second one will populate it with our 
-  demo dataset.
+* **Import Data**: Import some data into your cluster using our data import 
+  tool.
 
-* **Backups**: The "see my backups" will take you to the Backups tab, where 
-  you can see all your backups.
+* **See my backups**: CrateDB Cloud clusters can now be cloned to a new 
+  cluster from any backup.
```
Comment (SStorm) [inline docs/overview.rst:253]: I think this was a a pretty good explanation previously? Maybe add the clone sentence at the end?

```suggestion
* **See my backups**: The "see my backups" will take you to the Backups tab, where 
  you can see all your backups. CrateDB Cloud clusters can now be cloned to a new 
  cluster from any backup.
```

**Review by SStorm — APPROVED** [review]
:+1: A few minor wording suggestions, please feel free to adapt

**Review by kojinkai — APPROVED** [review]

**Pull request merged**